python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -252,7 +252,7 @@ def __init__(self, name, doc, dirty_flush_counter=10, update_index_time=30, dirt
         self._dirty_max_timeout = dirty_max_timeout
         self._dirty_min_timeout = dirty_min_timeout
 
-
+        ## Id's id(obj) of objects undergoing a transaction such as flush, remove, add, etc.
```

Comment:
PEP8 specifies a single `#` for line comments. No need to change it here but for future...